src/base/static/components/input-form/index.js

```diff
@@ -290,10 +290,11 @@ class InputForm extends Component {
   onSubmit() {
     Util.log("USER", "new-place", "submit-place-btn-click");
 
-    this.validateForm(this.createPlace);
+    Mixpanel.track("Clicked place form submit");
+    this.validateForm(this.submitForm);
   }
 
-  createPlace = async () => {
+  submitForm = async () => {
```

Comment:
I renamed this to avoid confusion between `this.createPlace` and `this.props.createPlace`.